src/pcapi/emails/new_offer_validation.py

```diff
@@ -14,3 +14,14 @@ def retrieve_data_for_offer_approval_email(offer: Offer) -> Dict:
             "pc_pro_offer_link": build_pc_pro_offer_link(offer),
         },
     }
+
+
+def retrieve_data_for_offer_rejection_email(offer: Offer) -> Dict:
+    return {
+        "MJ-TemplateID": 2613942,
+        "MJ-TemplateLanguage": True,
+        "Vars": {
+            "offer_name": offer.name,
+            "venue_name": offer.venue.publicName if offer.venue.publicName else offer.venue.name,
+        },
```

Comment:
```suggestion
            "venue_name": offer.venue.publicName if offer.venue.publicName else offer.venue.name,
            "pc_pro_offer_link": build_pc_pro_offer_link(offer),
        },
```

En renommant la variable `lien_offre_pcpro` du template MJ.

Reply:
bien vu celle du ticket d'avant sont en anglais aussi